test/updatejson.sh

```diff
@@ -2,9 +2,8 @@
 
 TOKEN=${1}
 
-curl --request GET --get "https://api.metadataapi.net/scenes?q=dorcelclub-2021-12-23-peeping-tom"     --header "Authorization: Bearer ${TOKEN}" --header "Content-Type: application/json"     --header "Accept: application/json" | jq > ./dc.json
-curl --request GET --get "https://api.metadataapi.net/scenes?q=evil-angel-2022-01-03-carmela-clutch-fabulous-anal-3-way"     --header "Authorization: Bearer ${TOKEN}" --header "Content-Type: application/json"     --header "Accept: application/json" | jq > ./ea.json
-curl --request GET --get "https://api.metadataapi.net/scenes/1678283"     --header "Authorization: Bearer ${TOKEN}" --header "Content-Type: application/json"     --header "Accept: application/json" | jq > ./ea.full.json
-curl --request GET --get "https://api.metadataapi.net/scenes?q=brazzers-exxtra-suck-suck-blow"     --header "Authorization: Bearer ${TOKEN}" --header "Content-Type: application/json"     --header "Accept: application/json" | jq > ./ssb2.json
-curl --request GET --get "https://api.metadataapi.net/scenes?q=brazzers-exxtra-suck-suck-blow"     --header "Authorization: Bearer ${TOKEN}" --header "Content-Type: application/json"     --header "Accept: application/json" | jq > ./ssb2.json
-curl --request GET --get "https://api.metadataapi.net/movies?q=petite18.Harper%20Red&limit=25"     --header "Authorization: Bearer ${TOKEN}" --header "Content-Type: application/json"     --header "Accept: application/json" | jq > ./p18.json
```

Comment:
these are more readable by human devs comparing test failures to data with jq in the mix

Reply:
I prefer have real server response, if you want human readable, then better use software which much faster that regular notepad for json like, [Dadroit JSON Viewer](https://dadroit.com)

Reply:
Fair enough, you do you :D

I'll look in to vs code json viewers